### ISSUE TITLE
Release plan

### ISSUE THREAD

**Issue by arokettu** [arokettu/php-random-polyfill#4]
Postpone final release until PHP 8.2 final release

* [x] RC 1 compatibility (done in beta 1)
* [x] RC 2 compatibility (done in rc 1)
* [x] RC 3 compatibility
* [x] RC 4 compatibility
* [x] RC 5 compatibility
* [x] RC 6 compatibility (done by #5 in 1.0.1)
* [x] RC 7 compatibility
* [x] Release compatibility

**Comment by TimWolla** [comment]
FYI: php/php-src#9839. Not yet merged due to one pending review, but this will become part of RC 6.

**Comment by arokettu** [comment]
@TimWolla Thanks. It seems it doesn't affect the object behavior and my tests still pass

**Comment by TimWolla** [comment]
The output for the following will change to match PHP 7.3 to 8.1:

```php
<?php

mt_srand(1, MT_RAND_PHP);

var_dump(str_shuffle('0123456789'));

if (class_exists(Random\Randomizer::class)) {
    $r = new Random\Randomizer(new Random\Engine\Mt19937(1, MT_RAND_PHP));
    var_dump($r->shuffleBytes('0123456789'));
}
```

https://3v4l.org/v8jFO

see:
```
$ g rev-parse HEAD
b9bee155a9c68a3c4ba70b9255a3aea5ee2570bc
$ make -j4 >/dev/null 2>&1
$ sapi/cli/php test.php
string(10) "8926013475"
string(10) "8926013475"
$ g co HEAD^
HEAD is now at 1ba80ff95c Merge branch 'PHP-8.1' into PHP-8.2
$ g rev-parse HEAD
1ba80ff95c926389be836c4cb82982fb2b578b13
$ make -j4 >/dev/null 2>&1
$ sapi/cli/php test.php
string(10) "8132476905"
string(10) "8132476905"
```

**Comment by TimWolla** [comment]
@arokettu I've just merged the previously linked PR.

**Comment by arokettu** [comment]
Ok, I see. I fixed it in #5, I will merge it once RC6 is out
